data/seeds/001-projects.js

```diff
@@ -0,0 +1,13 @@
+
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
+  return knex('projects').truncate()
+    .then(function () {
+      // Inserts seed entries
+      return knex('projects').insert([
+        { name: 'CS10', description : 'proj1 '  },
```

Comment:
Spaces and indentation could be made slightly more consistent here.  Nitpicky, I know, but it adds up.